app/api/search/elasticTypes.ts

```diff
@@ -1,5 +1,6 @@
 import { RequestParams } from '@elastic/elasticsearch';
 import { RequestBody } from '@elastic/elasticsearch/lib/Transport';
+import { Aggregations } from 'shared/types/Aggregations.d.ts';
```

Comment:
I don't think the extensions are required?

Reply:
it was not required, fixed, weird in my initial implementation i was getting an error.